src/components/Modals/CreateMultipleItemsModal/CreateMultipleItemsModal.container.ts

```diff
@@ -0,0 +1,32 @@
+import { connect } from 'react-redux'
+import { RootState } from 'modules/common/types'
+import { cancelSaveMultipleItems, saveMultipleItemsRequest, clearStateSaveMultipleItems } from 'modules/item/actions'
+import { getSavedItemsFiles, getMultipleItemsSaveState, getProgress } from 'modules/ui/createMultipleItems/selectors'
+import { getError } from 'modules/item/selectors'
+import { Collection, CollectionType } from 'modules/collection/types'
+import { getCollection } from 'modules/collection/selectors'
+import { getCollectionType } from 'modules/collection/utils'
+import { MapStateProps, MapDispatchProps, MapDispatch, OwnProps } from './CreateMultipleItemsModal.types'
+import CreateMultipleItemsModal from './CreateMultipleItemsModal'
+
+const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
+  const collection: Collection | null = ownProps.metadata.collectionId ? getCollection(state, ownProps.metadata.collectionId) : null
+  const collectionUrn: string | null =
+    collection?.urn && getCollectionType(collection) === CollectionType.THIRD_PARTY ? collection.urn : null
```

Comment:
I feel like this should be a method inside the component. My reasoning is that both collection types have URNs and there's no reason for a dev reading `collectionUrn` inside the component to derive that we're ONLY talking about TP URNs.
What do you think?